src/idpi/operators/lateral_operators.py

```diff
@@ -0,0 +1,192 @@
+"""Lateral operators."""
+
+# Standard library
+from typing import Literal
+
+# Third-party
+import numpy as np
+import xarray as xr
+
+
+def compute_weights(
```

Comment:
perhaps we could unittest this function ? for some of the use cases and compare against a numpy array of the expected values

Reply:
I mean an isolated test, out of the regression that is using fieldextra